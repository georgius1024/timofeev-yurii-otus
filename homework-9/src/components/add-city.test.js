import React from 'react'
import jest from 'jest-mock'
import renderer from 'react-test-renderer'
import AddCity from './add-city'
describe('AddCity', () => {
  it('renders correctly', () => {
    const onClose = jest.fn()
    const onSubmit = jest.fn()
    const DB = []
    const DOM = renderer.create(
      <AddCity active={true} onClose={onClose} onSubmit={onSubmit} DB={DB}/>
    )
    const html = JSON.stringify(DOM, null, 2)
    expect(DOM).toMatchSnapshot()
  })
})
