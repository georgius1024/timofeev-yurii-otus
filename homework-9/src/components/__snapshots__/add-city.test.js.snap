// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddCity renders correctly 1`] = `
<div
  className="modal  is-active"
>
  <div
    className="modal-background"
  />
  <div
    className="modal-content has-background-white has-text-black-ter form"
  >
    <div
      className="field"
    >
      <label
        className="label"
      >
        Add city
      </label>
      <div
        className="control"
      >
        <input
          className="input"
          onChange={[Function]}
          placeholder="Enter city name"
          type="text"
          value=""
        />
        <div
          className="typeahead"
        />
      </div>
    </div>
  </div>
  <button
    className="modal-close is-large"
    onClick={[Function]}
  />
</div>
`;
